### ISSUE TITLE
Mark spam messages in Unified Inbox

### ISSUE THREAD

**Issue by kurzer90** [k9mail/k-9#5676]
Sometimes spam message are still important messages, therefore I integrated my spam folder to the unified inbox.

Unfortunately, it is not obvious which message is originally located at the spam folder. 

It would be a good solution to set an icon in front of the message to mark it as spam in the unified inbox.

Thx

**Comment by life777eternal** [comment]
The spam folder is a separate folder than any Inbox folder. Plus we'd have to login to the email webmail and check the spam folder to mark the ones we want as not spam or add their email addresses to the contacts on the webmail. That way they don't keep getting marked as spam by the email provider.

**Comment by kurzer90** [comment]
Thanks so far, but I'm not sure if you getting me right.
> Plus we'd have to login to the email webmail and check the spam folder to mark the ones we want as not spam...

I do not ask for marking mails as "not spam".

> The spam folder is a separate folder than any Inbox folder...

I got this, but let's say I have 4 accounts. Checking all four Spam-Folders is kind of annoying and time consuming. 
So I integrated all the Spam-Folder(s) to the unified Inbox. In my understanding, this is a process done by K9, not the email provider. 
Achieved using the following procedure:

Manage Folders --> Choose Spam Folder --> Check the "Unify" box

This folder is also marked as a Spam-Folder considering the "Account settings".

The idea is to highlight all mails in the "Unified Inbox", which are originally stored in one of these Spam-Folders.
So I can see at first glance - having a look at the unified Inbox - which mails are considered as spam.

Does this explanation make more sense?

**Comment by life777eternal** [comment]
Ok, aye I see that. And I tried that for the spam folder on one of my email accounts. Every email from the spam folder showing in the "Unified Inbox" already has "*** SPAM ***" added to the beginning of the subject line. Plus we can set a color for each email account which shows a distinction of which email account the emails in the "Unified Inbox" are from.

**Comment by kurzer90** [comment]
> Ok, aye I see that. And I tried that for the spam folder on one of my email accounts. Every email from the spam folder showing in the "Unified Inbox" already has "*** SPAM ***" added to the beginning of the subject line.

Ok, but then it isn't working, at least in my case. 
Are you sure that the "***SPAM***" isn't something your mail provider or some spam filter provides? Otherwise this means that you're able to manipulate the subject of an email, right? I hope that isn't the case.

> Plus we can set a color for each email account which shows a distinction of which email account the emails in the "Unified Inbox" are from.

That I already noticed and is working great.

**Comment by life777eternal** [comment]
That "*** SPAM ***" added to the beginning of the subject line is done by my email provider, aye.  So an option to highlight the messages in the "Unified Inbox" from any Spam folder would be a handy addition.

**Comment by kurzer90** [comment]
> That "*** SPAM ***" added to the beginning of the subject line is done by my email provider, aye. So an option to highlight the messages in the "Unified Inbox" from any Spam folder would be a handy addition.

That's right and the reason I opened a request (issue) for an enhancement =)

I would appreciate this feature.

I would suggest showing this messages with a grey-redish (rgb 166, 89, 89) background (such as unread messages are shown in light-grey), as red is a warning signal.
